### PR TITLE
Printable oxy tanks

### PR DESCRIPTION
**Pull request by JoeHammad1844**
ive been sitting on this so longa ton of stuff has been added that just arent in this yml so I will need to manually add them

## About the PR

adds emergency, extended emergency and double emergency oxygen tanks to the autolathe

by the way ive been sitting on the code for this so long that a bunch of stuff thats been added isnt in the YML yet so i need to quickly manually re add them

**Media**

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**

:cl: JoeHammad
- add: The dark secrets of creating oxygen tanks has been revealed to the cargo department

### CODE REVIEW

**Comment by deltanedas** [comment]
theres no reason to print smaller ones when its just a few steel more for the vastly superior option
also cl is wrong sci hss an autolathe and sometimes engi

File: Resources/Prototypes/Entities/Structures/Machines/lathe.yml [hunk 1]
```diff
@@ -97,8 +97,11 @@
       - TRayScanner
       - GasAnalyzer
       - UtilityBelt
+      - EmergencyOxygenTank
+      - ExtendedEmergencyOxygenTank
+      - DoubleEmergencyOxygenTank
       - Pickaxe
-      - ModularReceiver
```
Comment (deltanedas) [inline Resources/Prototypes/Entities/Structures/Machines/lathe.yml:101]: i before e except after c

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
also webedit ops :trollface:

**Comment by JoeHammad1844** [comment]
what the fuck is an editweb

**Comment by JoeHammad1844** [comment]
image
hermm

**Comment by JoeHammad1844** [comment]
null value ops

**Comment by JoeHammad1844** [comment]
ss14 version of john howard: captain goes for a space walk and disappears

**Comment by metalgearsloth** [comment]
> ss14 version of john howard: captain goes for a space walk and disappears

Are you thinking of Harold Holt?

**Comment by deltanedas** [comment]
((gaming refren)

**Comment by JoeHammad1844** [comment]
yeah probably

**Comment by JoeHammad1844** [comment]
image
zamn okay, what does this mean?

**Comment by deltanedas** [comment]
ignore

**Comment by JoeHammad1844** [comment]
Odd

**Comment by UbaserB** [comment]
> theres no reason to print smaller ones when its just a few steel more for the vastly superior option
> also cl is wrong sci hss an autolathe and sometimes engi

you forgot about spiders

**Comment by deltanedas** [comment]
yes and spiders will print double emergency tanks theres no reason to print smaller ones

**Comment by JoeHammad1844** [comment]
you're probably right but it just feels wrong not to, you know? someone might just like one aesthetically, I get your point, but I also don't see a reason why not to include it

**Comment by KaiShibaa** [comment]
People would powergame and only grab the double emergency ones because it makes the other two redundant.

There's a reason they're rarely mapped.

**Comment by EmoGarbage404** [comment]
it seems weird that all of these are directly available at any autolathe.

Even if a tank has double capacity, it should have a cost that is greater than double since it has the added advantage of only taking up a single inventory slot.

Additionally, I think printing these is very bloat-y considering the autolathe is already full of random shit and oxygen tanks are extremely plentiful.

quite frankly, this just looks like your other prs trying to give salvage double extended emergency tanks but in another set of clothes.

**Comment by JoeHammad1844** [comment]
not entirely
you can just go down to atmos and ask for a bigger oxygen tank and get it for free, they might even fill it up more for you.
the ones that you can print start empty but you could just find oxygen tanks in maints, assuming cargo didn't already steal them all

also, I was trying to give salv the single extended tank not the double extended tank

**Comment by EmoGarbage404** [comment]
Considering these items are direct upgrades, they should not be available roundstart in an easily accessible / replenishable manner.

We've additionally rejected several other PRs which sought to similarly make these items more easily acquired.

Due to the combined benefits of having a significantly higher capacity while simultaneously fitting in a pocket, these should not be common or able to produced in any way.

closing this.

**Pull request closed without merging**